templates/octavia/config/octavia.conf

```diff
@@ -0,0 +1,110 @@
+[DEFAULT]
+debug=True
+# debug={{.Debug}}
+# transport_url={{.RabbitTransportURL}}
+rpc_response_timeout=60
+log_file=/var/log/octavia/octavia.log
+log_dir=/var/log/octavia
+[api_settings]
+bind_host=192.168.1.147
+bind_port=9876
+auth_strategy=keystone
+# enabled_provider_drivers=amphora: The Octavia Amphora driver.,octavia: Deprecated alias of the Octavia Amphora driver.
+healthcheck_enabled=True
+tls_cipher_prohibit_list=
+default_listener_tls_versions=TLSv1.2,TLSv1.3
+default_pool_tls_versions=TLSv1.2,TLSv1.3
+[database]
+[health_manager]
+health_update_threads=4
+stats_update_threads=4
+# heartbeat_key=FIXMEkey1
+[keystone_authtoken]
+www_authenticate_uri={{ .KeystonePublicURL }}
+auth_url={{ .KeystonePublicURL }}
+username={{ .ServiceUser }}
+# password=FIXMEpw3
+project_name=service
+project_domain_name=Default
+user_domain_name=Default
+auth_type=password
+# memcache_use_advanced_pool=True
+# memcached_servers=FIXMEhost1:11211
+# region_name=regionOne
+# interface=internal
+[certificates]
```

Comment:
We are moving to the dual CA model here right?
That is long standing tech debt in tripleo

Reply:
Yes.